packages/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -303,14 +309,18 @@ public void onResult(@NonNull Status status) {
     private void onSignInResult(GoogleSignInResult result) {
       if (result.isSuccess()) {
         GoogleSignInAccount account = result.getSignInAccount();
+        currentAccount = account;
         Map<String, Object> response = new HashMap<>();
-        response.put("displayName", account.getDisplayName());
         response.put("email", account.getEmail());
         response.put("id", account.getId());
-        response.put("idToken", account.getIdToken());
-        Uri photoUrl = account.getPhotoUrl();
-        if (photoUrl != null) {
-          response.put("photoUrl", photoUrl.toString());
+        if (account.getIdToken() != null) {
```

Comment:
An internal code review made me think that maybe it was a good idea, but there's no real reason, so I reverted it here.